sources/packages/web/src/components/common/students/StudentProfile.vue

```diff
@@ -98,8 +98,15 @@
 import { onMounted, ref, defineComponent } from "vue";
 import { StudentService } from "@/services/StudentService";
 import { useFormatters } from "@/composables";
-import { StudentProfile } from "@/types";
 import { AddressAPIOutDTO } from "@/services/http/dto";
+import { StudentProfile } from "@/types";
+
+/**
+ *  Used to combine institution and ministry DTOs and make SIN explicitly mandatory.
+ */
+interface SharedStudentProfile extends Omit<StudentProfile, "sin"> {
```

Comment:
What exactly are we trying to do here?

Reply:
That's an interface to represent institution and ministry properties as we can have both here in this case.